src/R/Components/Impl/PackageManager/Implementation/ViewModel/RPackageManagerViewModel.cs

```diff
@@ -68,6 +70,16 @@ internal class RPackageManagerViewModel : BindableBase, IRPackageManagerViewMode
             private set { SetProperty(ref _isLoading, value); }
         }
 
+        public string FirstError {
+            get { return _firstError; }
+            private set { SetProperty(ref _firstError, value); }
+        }
+
+        public bool HasMultipleErrors {
```

Comment:
Would it be easier to make this a computed property (based on `_errorMessages.Count`)?